### PR TITLE
detection probability per detection site and state

### PR DESCRIPTION
**Pull request by michaeleh**

### CODE REVIEW

File: src/corona_hakab_model/consts.py [hunk 1]
```diff
@@ -27,6 +27,20 @@
 
 # TODO split into a couple of files. one for each aspect of the simulation
 class Consts(NamedTuple):
+    # medical states
+    LATENT = "Latent"
```
Comment (bgmoshe) [inline src/corona_hakab_model/consts.py:31]: Why not make this just an enum?

File: src/corona_hakab_model/detection_model/healthcare.py [hunk 2]
```diff
@@ -26,17 +27,13 @@ class PendingTestResults(Queue[PendingTestResult]):
 
 
 class DetectionTest:
-    def __init__(self, detection_prob, false_alarm_prob, time_until_result):
-        self.detection_prob = detection_prob
-        self.false_alarm_prob = false_alarm_prob
+    def __init__(self, state_to_detection_prop: Dict[MedicalState, float], time_until_result):
+        self.state_to_detection_prop = state_to_detection_prop
```
Comment (bgmoshe) [inline src/corona_hakab_model/detection_model/healthcare.py]: It is more appropriate to copy the dict, no? Dictionary is mutable.

**Review by bgmoshe — COMMENTED** [review]
Some small changes

File: src/corona_hakab_model/consts.py (hunk 1 above)
Comment (RoyIsland) [inline src/corona_hakab_model/consts.py:31]: add those variables to simulation_parameters_example.py

File: src/corona_hakab_model/consts.py [hunk 3]
```diff
@@ -105,9 +119,21 @@ class Consts(NamedTuple):
     detection_pool: List[DetectionSettings] = [
         DetectionSettings(
             name="hospital",
-            detection_test=DetectionTest(detection_prob=0.98,
-                                         false_alarm_prob=0.,
-                                         time_until_result=3),
+            detection_test=DetectionTest({
```
Comment (ghost) [inline src/corona_hakab_model/consts.py:122]: add those variables changes to simulation_parameters_example.py

**Review by bgmoshe — APPROVED** [review]

File: src/corona_hakab_model/consts.py (hunk 3 above)
Comment (michaeleh) [reply src/corona_hakab_model/consts.py:122]: done

**Pull request merged**